### ISSUE TITLE
Date limit facet code is producing duplicate IDs

### ISSUE THREAD

**Issue by erinchambers** [CDRH/orchid#322]
The `<input type="text">` elements for "From Month" and "From Day" have identical IDs, `id="date_from_"`. Is it possible we are missing some code that makes those unique from each other?

**Comment by erinchambers** [comment]
Fixed
